src/pages/SubmissionConfig.js

```diff
@@ -360,7 +366,22 @@ export const SubmissionConfig = ({ methodId }) => {
           ]),
           div({ style: { lineHeight: 2.0, marginTop: '1.5rem' } }, [
             div([h(TextCell, ['This will launch ', span({ style: { fontWeight: 'bold' } }, [_.keys(selectedRecords).length]), ' workflow(s).'])]),
-            h(TextCell, { style: { marginTop: '1rem' } }, ['Running workflows will generate cloud compute charges.'])
+            h(TextCell, { style: { marginTop: '1rem' } }, ['Running workflows will generate cloud compute charges.']),
+            ...Utils.cond([workflowSubmissionError, () => [
```

Comment:
small nit: Instead of using `Utils.cond` you can also simply do
``` 
workflowSubmissionError && div([
 div({ style: { display: 'flex', alignItems: 'center', marginTop: '1rem' } }, [ 
...
])
```
This will display the error message div if `workflowSubmissionError` is not undefined or null.